core/controllers/domain_objects_validator.py

```diff
@@ -243,3 +243,49 @@ def validate_suggestion_images(files):
     # The files argument do not represent any domain class, hence dict form
     # of the data is returned from here.
     return files
+
+
+def validate_exploration_change(exp_change_dict):
+    """Validate the exploration change list
+
+    Args:
+        exp_change_dict: dict. The exploration change dictionary
+            that needs to be validated.
+
+    Returns:
+        exp_domain.ExplorationChange. The exploration change domain
+        object.
+    """
+    if exp_change_dict.get('cmd') is None:
+        raise base.BaseHandler.InvalidInputException(
+            'Missing cmd key in change dict')
+
+    exp_change_commands = [
```

Comment:
valid_exp_change_commands or allowed_exp_change_commands (to prevent readers thinking that this relates to the dict that is being passed in)

Reply:
Done